tests/topotests/bgp_bfd_down_cease_notification/test_bgp_bfd_down_cease_notification_shutdown.py

```diff
@@ -116,6 +116,18 @@ def _bgp_bfd_down_notification():
     _, result = topotest.run_and_expect(test_func, None, count=30, wait=1)
     assert result is None, "Failed to see BGP Cease/BFD Down notification message on R2"
 
+    def _bgp_status_post_bfd_profile_shutdown():
```

Comment:
Isn't this kind of backwards?  You want to test that the idle state has been around for at least 10 seconds.   This does not guarantee that we have stayed in the idle state at all!  All it does is show that we are in the idle state after 10 seconds.  While I maybe arguing semantics here a bit I think it matters.  Unfortunately the bgp neighbor show command does not give you Idle time.  I would recommend that we add a IdleTimer similiar in nature to the bgpTimerUpMsec that gives you how long you want the idle time to reach.  Then you can just do a regular run and expect to get the idle timer up over 10 seconds.

Reply:
We already have `peerUptimeMsec` which is fine to use together with `state`. I changed the test.